### PR TITLE
Updates for Token Metadata V1.3

### PR DESCRIPTION
**Pull request by samuelvanderwaal**
Documentation updates for Token Metadata V1.3. Add new instructions:

* set_collection_size
* set_token_standard
* burn_nft
* verify_sized_collection_item
* unverify_sized_collection_item
* set_and_verify_sized_collection_item
* create_metadata_account_v3

V1.3 [overview](https://github.com/metaplex-foundation/metaplex-program-library/pull/459) and [full proposal](https://github.com/metaplex-foundation/metaplex-program-library/discussions/444)

### CODE REVIEW

File: docs/programs/token-metadata/changelog/v1.3.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+## Deprecations
+
+- The following instruction now shows a deprecation warning when executed
+  - `CreateMetadataAccountV2` - _please use `CreateMetadataAccountV3` instead_
+
+## Removals
+
+Removal of previously deprecated instructions
+
+-
```
Comment (lorisleiva) [inline docs/programs/token-metadata/changelog/v1.3.md:41]: There's an empty bullet point here. Were you planning on listing the deprecated instructions that have been removed as of v1.3?

**Review by lorisleiva — APPROVED** [review]
That's great, thank you for that!

I've made some changes in the text and in the diagrams to stay consistent with what we had before but, otherwise, all good. 👌

I've also added a little comment below in case it was forgotten.

File: docs/programs/token-metadata/changelog/v1.3.md (hunk 1 above)
Comment (samuelvanderwaal) [reply docs/programs/token-metadata/changelog/v1.3.md:41]: Yeah, exactly, good catch! I'll fix that.

**Comment by samuelvanderwaal** [comment]
This is waiting until v1.3 is deployed to `mainnet`.

**Comment by lorisleiva** [comment]
Yes! 🚀 
Thanks for the conflict resolution! ❤️

**Pull request merged**